pkg/networkservice/common/timeout/server.go

```diff
@@ -97,3 +92,34 @@ func (s *timeoutServer) Close(ctx context.Context, conn *networkservice.Connecti
 	}
 	return &empty.Empty{}, err
 }
+
+func after(ctx context.Context, conn *networkservice.Connection) time.Duration {
```

Comment:
I think we need to rename it. 

For example,

```suggestion
func minTokenTimeout(ctx context.Context, conn *networkservice.Connection) time.Duration {
```